pom.xml

```diff
@@ -147,6 +151,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
```

Comment:
by adding this and keeping the findbugs-maven-plugin (just above), are now running both FindBugs and SpotBugs for our own code in this project (not for the users installing these plugins). That seems perhaps a bit excessive, to me, and IMHO we should ditch the findbugs-maven-plugin here with this change, what do you think?

Reply:
I aggree and will change that